### DISCUSSION TITLE
Custom CSS (Dark Turquoise)

### DISCUSSION THREAD

**Discussion by optimvsjag** [orestbida/cookieconsent#664] [Q&A] [answered]
Hi team, I'm trying to set the Dark Turquoise theme and I can see the results in the playground, but when I config my site not show this theme. Only show the deafult light theme.

I didn't find if I need to add some aditional css code and where need to add it to show this theme correctly. Anyone can show me how can I do it?

![image](https://github.com/orestbida/cookieconsent/assets/71232507/7b42a1f5-8f95-43e4-9966-d0bd51f279a9)

Thanks a lot.

**Comment by orestbida** [comment, marked as answer]
You need to copy the css code in *Dark Turquoise (CSS) expandable section*, as is, and place it after the plugin's css. Then you need to add the class for the theme:
```js
import 'https://cdn.jsdelivr.net/gh/orestbida/cookieconsent@3.0.0/dist/cookieconsent.umd.js';

// toggle dark mode + custom theme
document.documentElement.classList.add('cc--darkmode', 'cc--dark-turquoise');

CookieConsent.run({ ... });
```
